packages/flutter/lib/src/material/chip_theme.dart

```diff
@@ -299,6 +301,17 @@ class ChipThemeData extends Diagnosticable {
   /// The chip is selected when [selected] is true.
   final Color secondarySelectedColor;
 
+  /// Color of the chip's shadow when the elevation is greater than 0.
+  ///
+  /// If null, the chip defaults to [Colors.black].
+  final Color shadowColor;
```

Comment:
This comment should either weave in a reference to [selectedShadowColor] or include a see also section.

I imagine that typically developers that set this will also set selectedShadowColor.

Reply:
Good idea, done